design/cstor/v1/cstor_volume.md

```diff
@@ -0,0 +1,353 @@
+# Move `CStorVolumes`, `CStorVolumeReplica` resources Schema to version `v1`
```

Comment:
@prateekpandey14, can you follow the same format as the cstorpool.md. 

This document should just contain the new v1 API schema. No need to mention the old schema or changes from the earlier schema is this PR. 

Same comments apply to the other file as well.

Reply:
Add the schema for cStorVolumeAttachment as well.

Reply:
done